kong/plugins/datadog/schema.lua

```diff
@@ -114,31 +90,12 @@ return {
                   { consumer_identifier = { type = "string", one_of = CONSUMER_IDENTIFIERS }, },
```

Comment:
Always adding a consumer id increases the size of the metric transmitted over to datadog. Consumer_identifier is still optional, in case you just want the metric irrespective of the caller, for instance latency.